backend/tests/manual/test-rawsql-export-import.sh

```
@@ -63,17 +63,17 @@ exit_with_status() {
 API_BASE="$TEST_API_BASE"
 AUTH_TOKEN=""
 
-# Read .env file to get admin credentials
+# Read .env file to get root admin credentials
 if [ -f "$SCRIPT_DIR/../../.env" ]; then
     export $(grep -v '^#' "$SCRIPT_DIR/../../.env" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)